Adding Docs for Cody Ignore (#77)
sourcegraph/docs: docs/cody/capabilities/ignore-context.mdx

# Cody Ignore

> <Callout type="note">This feature is currently in the Experimental stage.</Callout>

Reviewer: ```suggestion
<Callout type="note">This feature is currently in the Experimental stage and only supported with VS Code extension.</Callout>
```

## Enabling Unstable Features

To use the ignore context feature, you first need to enable unstable features in Cody. Here's how:

1. Open your settings in Cody Extension
2. Open `settings.json` link in the settings
3. Add a new line: `"cody.unstableFeatures": true`.

> This will enable the use of experimental features, including the ignore context feature.

Reviewer: ```suggestion
This will enable experimental features, including the ignore context feature.
```

## Using the Ignore Context Feature

> To ignore certain files or folders from the context taken by Cody, you need to create a `ignore` file in your project. Here's how:

Reviewer: ```suggestion
To ignore specific files or folders from the context taken by Cody, you need to create a `ignore` file in your project. Here's how:
```

1. Create a new folder in your project root named `.cody`.
2. Inside the `.cody` folder, create a file named `ignore`.
3. In the `ignore` file, specify the files or folders you want to ignore.

> The `.codyignore` file works similarly to a `.gitignore` file. If you want to ignore a file named `secret.json`, you would add `secret.json` to your `.codyignore` file. If you want to ignore a folder named `lib/shared`, you would add `lib/shared/` to your `.codyignore` file.

Reviewer: ```suggestion
The `.codyignore` file works similarly to a `.gitignore` file. If you want to ignore a file named `secret.json`, you would add `secret.json` to your `.codyignore` file. If you're going to ignore a folder called `lib/shared`, you would add `lib/shared/` to your `.codyignore` file.
```

> Once a file or folder is added to the `.codyignore` file, Cody will no longer provide autocomplete suggestions for that file or folder, and it will not show up in your chat results.

Reviewer: ```suggestion
Once a file or folder is added to the `.codyignore` file, Cody will no longer provide autocomplete suggestions for that file or folder, and it will not appear in your chat results.
```

> Thank you for trying out this experimental feature, and we look forward to your feedback!

Reviewer: ```suggestion

```
